app/src/main/index.js

```diff
@@ -428,6 +444,11 @@ const eventHandlers = {
 
   "successful-launch": () => {
     console.log(`[START SUCCESS] Vue app successfuly started`)
+  },
+
+  createAccount: (event, name) => {
```

Comment:
would probably need name + seed?

Reply:
Yes.  The word "spike" refers to something that pierces as many layers of the software as possible, narrow instead of broad, so I deliberately sent only one piece of information that will ultimately be passed through.